peachjam/models/core_document_model.py

```diff
@@ -400,6 +400,9 @@ class AttachmentAbstractModel(models.Model):
     mimetype = models.CharField(_("mimetype"), max_length=1024, null=False, blank=False)
     size = models.BigIntegerField(_("size"), default=0)
     file = DynamicStorageFileField(_("file"), upload_to=file_location, max_length=1024)
+    source_url = models.URLField(
```

Comment:
My idea was to put this on SourceFile, not AttachmentAbstractModel. What's the benefit of having it here? What are the semantics of having a source_url for an image attachment, say?

Unless we have a good reason, I think we should limit this just to SourceFile, because we have a good reason for it there. We can always move it later if there is a reason to have it on all attachments.